src/common/images/imageGallery.component.tsx

```diff
@@ -399,7 +406,6 @@ const ImageGallery = (props: ImageGalleryProps) => {
                           row={card.row as MRT_Row<APIImage>}
                           table={table}
                           sx={{
-                            ariaLabel: `actions ${card.row.original.file_name} photo button`,
```

Comment:
This aria-label wasn't being used. I was use the one from the table localization